src/emulator/functionsEmulator.ts

```diff
@@ -879,10 +876,16 @@ export class FunctionsEmulator implements EmulatorInstance {
 
   async reloadTriggers() {
     this.triggerGeneration++;
-    return this.loadTriggers(true);
+    return this.loadTriggers(false);
```

Comment:
Boolean arguments are hard to understand, consider writing `this.loadTriggers(/* parameterName= */ false)`

Reply:
👍